docs/samples/calling/app.js

```diff
@@ -899,7 +918,7 @@ async function createVoiceMail() {
 
     try {
       if (window.voicemail === undefined) {
-        voicemail = window.voicemail = Calling.createVoicemailClient(webex, logger);
+        voicemail = window.voicemail = calling.voicemailClient(webex, logger);
```

Comment:
all the client instance should be created once and referred it globally on top of the file

Reply:
taken care